internal/provider/resource_user_test.go

```diff
@@ -187,6 +193,56 @@ func TestAccUser_existing_mac_deny(t *testing.T) {
 	})
 }
 
+func TestAccUser_fingerprint(t *testing.T) {
+	testMAC := generateTestMac()
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testCheckUserDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccUserConfig_fingerprint(testMAC, "tfacc", 123),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("unifi_user.test", "dev_id_override", "123"),
+				),
+			},
+			{
```

Comment:
You should add the `importStep` in here to test roundtripping the information.

Reply:
I missed your comment somehow. I'm not quite sure I follow, what does `importStep` do?

Reply:
Sorry @joshuaspence forgot to respond here! `importStep` tells the test framework to drop the state, re-import the resource using just its ID, and compare the imported state to the previous state to ensure no diff (basically ensuring a clean read is the same as config+update, etc.).

Reply:
Got it, that makes sense. Seems like something we should do for all test resources. I have a few weeks leave from work coming up, will likely finish this off then.